src/note.rs

```diff
@@ -261,6 +261,12 @@ impl Note {
         &self.value_commitment
     }
 
+    /// Returns the encryped data from the note as an array of
+    /// BlsScalar's
```

Comment:
This should be more like `Returns the encrypted data's cipher` or something like that.